commons/connector/fitbit/fitbit_intraday_heart_rate_variability.avsc

```diff
@@ -0,0 +1,14 @@
+{
+  "namespace": "org.radarcns.connector.fitbit",
+  "type": "record",
+  "name": "FitbitIntradayHeartRateVariability",
+  "doc": "Intra day heart rate data from fitbit device.",
```

Comment:
Can the doc be updated with info from fitbit webAPI docs?

Reply:
Alright, will do.